src/bundle/Resources/views/themes/admin/trash/list.html.twig

```diff
@@ -19,7 +23,64 @@
         <div class="px-0 ez-content-container">
             <section class="container mt-5">
                 {% include '@ezdesign/ui/page_title.html.twig' with { title: 'trash.headline'|trans|desc('Trash'), icon_name: 'trash' } %}
+                
+                {{ form_start(form_search, { attr: { class: 'ez-trash-search-form' } }) }}
+                    <div class="ez-trash-search-form__inputs-group px-4">
+                        {% set creator = form_search.vars.data ? form_search.vars.data.creator : '' %}
+
+                        {{ form_widget(form_search.content_type, {'attr': {'class': 'ez-trash-search-form__item ez-trash-search-form__item--auto-send'}}) }}
+                        {{ form_widget(form_search.section, {'attr': {'class': 'ez-trash-search-form__item ez-trash-search-form__item--auto-send'}}) }}
+                        
+                        <div class='ez-trash-search-form__item ez-trash-search-form__item--trashed'>
+                            {{ form_widget(form_search.trashed, {'attr': {'data-target-selector': '.ez-trash-search-form__range-wrapper--select-trashed-range'}}) }}
+                        </div>
+                        <div class="ez-trash-search-form__item ez-trash-search-form__item--creator">
+                            {% set creator = form_search.vars.data ? form_search.vars.data.creator : '' %}
```

Comment:
this looks like a duplicate https://github.com/ezsystems/ezplatform-admin-ui/pull/1408/files#diff-714e4b94f463ed0adb56ec62ada630e2R29